templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -136,3 +136,24 @@ a space before your phrase or submit it as a quote block by following
 the code syntax highlighting format.
 
 ![Quotes](/static/images/help/quotes-screenshot.png)
+
+## TeX math
+
+You can display TeX mathematical expressions in Zulip.
+[TeX](http://www.tug.org/interest.html#doc), amongst other things, allow you to
+easily write formulas through commands. Zulip's implementation of this
+typesetting is [KaTeX](https://github.com/Khan/KaTeX), and you can see the full
+set of  compatible commands
+[here](https://github.com/Khan/KaTeX/wiki/Function-Support-in-KaTeX).
+
+For inline content, surround
+valid TeX syntax with double dollar signs `$$`.
+
+![Inline TeX](/static/images/help/inline-tex-screenshot.png)
+
+Also, you can show TeX in *display mode*, which is centered and with
```

Comment:
"You can show expressions, such as expanded integrals, in TeX **display mode** to display them fully-sized in the center of messages by fencing them with three back-ticks (` ``` `) with `math`, tex`, or `latex` immediately after the first set of back-ticks.